fastly-wordpress-plugin.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -xeo pipefail
+
+cd /var/www/html
+
+wp plugin install fastly
+wp plugin activate fastly
+
+# wp fastly configset general fastly_api_key ${api_key}
```

Comment:
the `wp fastly configset general <key> <value>` commands work to overrwrite an existing value, but for some reason can't do the initial setting when the value is empty

Reply:
@vvuksan any idea why?

Reply:
filed a bug: https://github.com/fastly/WordPress-Plugin/issues/100